mirisuzanne/cascading-color-system
# Mock up radio-mode form controls

![sea lion pup](https://scentsofself.files.wordpress.com/2010/08/sea_lion_pup.jpg)


## [Trello card](https://trello.com/c/pRMyOBsR/14-i-want-cascading-colors-to-support-a-three-way-mode-toggle)

## Description



## Steps to test/reproduce
1. checkout the branch
2. open `demo/index.html` in a browser
3. all the old form controls should continue to work
4. [x] Make the mode radio toggles functional
5. [x] Make the `data-ccs-field~="unset-theme"` option functional

## Show me
![form with new radio inputs](https://dl.dropbox.com/s/qnpoj2gv42dzs96/Screenshot%202020-06-22%2016.19.47.jpg)
